frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1226,20 +1228,12 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                         if (!tab) {
                             throw new Error('Developer Error, tab not found')
                         }
-                        return urls.insightNew(
-                            { properties: webAnalyticsFilters, date_from: dateFrom, date_to: dateTo },
```

Comment:
@robbie-c I couldn't figure out what adding the properties here does. Any hints for me?

Reply:
It should copy the property filters from web analytics into the new insight

Reply:
Not sure if I understand this correctly, but property filters should already be part of the queries set on the new insight. Let me know if this breaks anything that I don't see right now and I'm happy to adapt accordingly.

Reply:
Ah you're right, looks like it wasn't necessary. I suspect I just passed these arguments in because they were there :) Just ran these changes locally and they worked fine, so all good from me